packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -153,6 +159,7 @@ function Cell(props) {
 
   const className = cx('worklist-data-grid-cell', {
     masked: isMasked,
+    pinned: columnIndex < columnContext.pinnedColumnsLength,
```

Comment:
If you have an isPinned prop, why would you need this calculation.  I would expect for the parent Row component to handle determining if the cell is pinned.

Reply:
The `isPinned` prop above is leftover code we forgot to remove. There is no `isPinned` prop.

Reply:
I think it makes sense for the property to exist.  It's parent component can pass the value to it.